### PR TITLE
Add support for default column clauses in show create table.

### PR DESCRIPTION
**Pull request by kovrus**
## Checklist

 - [ ] User relevant changes are recorded in ``CHANGES.txt``
 - [ ] Touched code is covered by tests
 - [ ] Documentation has been updated if necessary
 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [ ] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

File: sql/src/test/java/io/crate/integrationtests/ShowIntegrationTest.java [hunk 1]
```diff
@@ -239,6 +239,23 @@ public void testShowCreateTableWithGeneratedColumn() {
                   ")");
     }
 
+    @Test
+    public void testShowCreateTableWithDefaultClauseColumns() {
```
Comment (kovrus) [inline sql/src/test/java/io/crate/integrationtests/ShowIntegrationTest.java]: or shall we merge this test with generated columns and name it `testShowCreateTableWithColumnExpressions`?

Comment (smakalias) [reply sql/src/test/java/io/crate/integrationtests/ShowIntegrationTest.java]: I personally prefer a separate test.

Comment (mfussenegger) [reply sql/src/test/java/io/crate/integrationtests/ShowIntegrationTest.java]: We could probably also use the `SqlExecutor` infrastructure to create a test that is a bit cheaper than a full integration test.

Comment (kovrus) [reply sql/src/test/java/io/crate/integrationtests/ShowIntegrationTest.java]: replaced the `ShowIntegrationTest` test with the test for the `MetaDataToASTNodeResolver` that uses the SqlExecutor infrastructure.

**Pull request merged**